runtime/criusupport/criusupport.cpp

```diff
@@ -404,6 +460,20 @@ Java_org_eclipse_openj9_criu_CRIUSupport_checkpointJVMImpl(JNIEnv *env,
 
 		acquireSafeOrExcusiveVMAccess(currentThread, vmFuncs, safePoint);
 
+		for (UDATA i = 0; !checkIfSafeToCheckpoint(currentThread); i++) {
```

Comment:
For safety, perhaps add a 0 check on maxRetryForNotCheckpointSafe in case the value becomes variable in the future.